aip/0123.md

```diff
@@ -73,8 +74,64 @@ message Topic {
 - Pattern variables (the segments within braces) **must** use `snake_case`, and
   **must not** use an `_id` suffix.
 
+#### Complex resource ID path segments
+
+Note: Complex resource ID path segments are supported for historical reasons,
```

Comment:
"are supported" is a bit weird. Supported by what?
(I know you mean client libraries, but client libraries are not really a primary concern for this document.)

This can probably just be "Complex resource ID path segments generally **should not** be used...".

Reply:
Updated (now in aip-4231)